### PR TITLE
Update MAIN-SBP-AMD-EPYC-4-SLES15SP4.xml

### PR DESCRIPTION
**Pull request by dariavladykina**
This PR presents linguistic suggestions for the SBP copy.

### CODE REVIEW

File: xml/MAIN-SBP-AMD-EPYC-4-SLES15SP4.xml [hunk 1]
```diff
@@ -1203,22 +1203,22 @@ Setting cpu: 2
...
         >https://www.cs.virginia.edu/stream/</link>). It can be used to measure bandwidth
         of each cache level and bandwidth to main memory while calculating four basic
-        vector operations.  Each operation can exhibit different throughputs to main memory
+        vector operations. Each operation can exhibit different throughputs to main memory
         depending on the locations and type of access.</para>
 
       <para>The benchmark was configured to run both single-threaded and parallelized with
-	OpenMP to take advantage of multiple memory controllers. The array elements for
-	the benchmark was set at 268,435,456 elements at compile time so each that array
+	OpenMP to take advantage of multiple memory controllers. The array of elements for
```
Comment (chabowski) [inline xml/MAIN-SBP-AMD-EPYC-4-SLES15SP4.xml:1214]: `The array elements for the benchmark were set at 268,435,456 elements at compile time`

array element is a fix term

**Review by chabowski — CHANGES_REQUESTED** [review]
Thanks Daria, for the very thorough language review! I only found one tiny little thing that needs to be changed. Let me know if I should change it or if you would like to do it. 

Thought I had caught all "processors" in the context of AMD EPYC that had not been capitalized  - but seems I did forget to apply the changes LOL. Thanks again!

File: xml/MAIN-SBP-AMD-EPYC-4-SLES15SP4.xml (hunk 1 above)
Comment (dariavladykina) [inline xml/MAIN-SBP-AMD-EPYC-4-SLES15SP4.xml:1214]:
```suggestion
	OpenMP to take advantage of multiple memory controllers. The array elements for
```

**Review by dariavladykina — COMMENTED** [review]
Apply review suggestions

File: xml/MAIN-SBP-AMD-EPYC-4-SLES15SP4.xml [hunk 2]
```diff
@@ -1203,22 +1203,22 @@ Setting cpu: 2
...
+        created by Dr. John D. McCalpin from the University of Virginia (for more
         information, see <link xlink:href="https://www.cs.virginia.edu/stream/"
         >https://www.cs.virginia.edu/stream/</link>). It can be used to measure bandwidth
         of each cache level and bandwidth to main memory while calculating four basic
-        vector operations.  Each operation can exhibit different throughputs to main memory
+        vector operations. Each operation can exhibit different throughputs to main memory
         depending on the locations and type of access.</para>
 
       <para>The benchmark was configured to run both single-threaded and parallelized with
 	OpenMP to take advantage of multiple memory controllers. The array elements for
```
Comment (dariavladykina) [inline xml/MAIN-SBP-AMD-EPYC-4-SLES15SP4.xml:1214]:
```suggestion
	OpenMP to take advantage of multiple memory controllers. The array of elements for
```

**Review by chabowski — APPROVED** [review]

**Pull request merged**